hack/gen-flavors.sh

```diff
@@ -17,10 +17,17 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-#set -o verbose
+# set -o verbose
 
 root=$(dirname "${BASH_SOURCE[0]}")/..
+script_invocation_dir="$(pwd)"
 kustomize="${root}/hack/tools/bin/kustomize"
+
+# go to root and install kustomize if missing
+cd ${root} && make "${kustomize##*/}"
```

Comment:
```suggestion
make -C ${root} "${kustomize##*/}"
```
If you use `make -C <dir>` you can invoke it from a different directory.

Reply:
Thats a great suggestion, it makes shell script concise!

Reply:
Addressed via commit dc6d1f602487940b00cbd53b59f014d7fc4bf089

Reply:
I used `make --directory=DIRECTORY` instead of `-C DIRECTORY` for better readability.